robottelo/cli/contentview.py

```diff
@@ -158,3 +158,10 @@ def remove_from_environment(cls, options=None):
         """Remove content-view from an enviornment"""
         cls.command_sub = 'remove-from-environment'
         return cls.execute(cls._construct_command(options))
+
+    @classmethod
+    def remove(cls, options=None):
+        """Remove versions and/or environments from a content view and
+        reassign content hosts and keys"""
```

Comment:
Other are one line docstrings, and using `"""One line."""` is ok. If the docstring requires more than one line then it should adopt the format:

```python
"""First line
continuation of first line if needed

other lines

"""
```